### PR TITLE
Remove generated go.mod file in tests

### PR DESCRIPTION
**Pull request by ChloePlanet**

### CODE REVIEW

File: mod/module_test.go [hunk 1]
```diff
@@ -147,3 +150,22 @@ func TestHasPathPrefix(t *testing.T) {
 
 	assert.False(t, hasPathPrefix("github.com/anz-bank/sysl2", "github.com/anz-bank/sysl/deps.sysl"))
 }
+
+func removeFile(t *testing.T, fs afero.Fs, file string, isDir bool) {
+	if isDir {
+		exists, err := afero.DirExists(fs, file)
+		assert.NoError(t, err)
```
Comment (nofun97) [inline mod/module_test.go:157]: this should use `require.NoError` to make sure that the program stops if there's an error

Comment (nofun97) [reply mod/module_test.go:157]: any `assert.NoError` in this function should get the same changes

File: mod/module_test.go [hunk 2]
```diff
@@ -26,8 +26,9 @@ func TestConfigGitHubMode(t *testing.T) {
 }
 
 func TestConfigGoModulesMode(t *testing.T) {
-	fs := afero.NewMemMapFs()
+	fs := afero.NewOsFs()
```
Comment (nofun97) [inline mod/module_test.go:29]: can this use `MemMapFs` instead?

Comment (ChloePlanet) [reply mod/module_test.go:29]: nah this runs `go mod init` which will create `go.mod` file in os filesystem

**Review by nofun97 — APPROVED** [review]
LGTM

**Pull request merged**